cmake/PAL.cmake

```diff
@@ -298,21 +298,27 @@ function(ly_get_absolute_pal_filename out_name in_name)
 
     # parent relative path is optional
     if(${ARGC} GREATER 4)
-        set(parent_relative_path ${ARGV4})
+        if(ARGV4)
```

Comment:
I think you can just do `if(ARGV4)`, if ARGC is < 4 ARGV4 should be empty, if not, they can be done in the same line